### ISSUE TITLE
Create nf-tests for "fusioninspector" module

### ISSUE THREAD

**Issue by nschcolnicov** [nf-core/rnafusion#523]
### Description of feature

Create nf-tests for "fusioninspector" module

**Comment by atrigila** [comment]
Notes:
- Requires starfusion references

**Comment by atrigila** [comment]
On hold until we can have test starfusion references in s3

**Comment by atrigila** [comment]
Depends on #579

**Comment by atrigila** [comment]
Current container does not work #594 
Needs to be updated, but available version in bioconda is 2.8.0. We need to update it to 2.10.0.

**Comment by atrigila** [comment]
Closed via #601
